polyfills/Array/from/polyfill.js

```diff
@@ -1,104 +1,205 @@
+(function(global){
+	'use strict';
 
-// Wrapped in IIFE to prevent leaking to global scope.
-(function () {
-	function parseIterable (arraylike) {
+	function toObject(value) {
+		if (value == null) {
+			throw new TypeError('Cannot call method on ' + value);
+		}
+		return Object(value);
+	}
+	function toLength(argument) {
+		var MAX_SAFE_INTEGER = Math.pow(2, 53) - 1;
+		var number = Number(argument);
+		if (isNaN(number)) {
+			number = 0;
+		}
+		var len = (number >= 0 ? 1 : -1) * Math.floor(Math.abs(number));
+		if (len <= 0) { return 0; } // includes converting -0 to +0
+		if (len > MAX_SAFE_INTEGER) { return MAX_SAFE_INTEGER; }
+		return len;
+	}
+	function isCallable(value) {
+		return typeof value === 'function';
+	};
+	function isString (value) {
+		return typeof value === 'string';
```

Comment:
This check need to accept `Object('str')` and similar objects. https://github.com/ljharb/is-string